src/Cassandra.IntegrationTests/Linq/LinqMethods/Count.cs

```diff
@@ -66,6 +83,18 @@ public void LinqCount_Where_Async()
             Assert.AreEqual(1, count);
         }
 
+        #pragma warning disable 618
```

Comment:
Instead of declaring a new entity, reuse one if possible and define mappings per tests using the fluent mapping (not the attributes).

On the other hard, we should stop using Linq attributes and use `Cassandra.Mapping.Attributes` instead.